modules/schat.cpp

```diff
@@ -226,9 +230,15 @@ class CSChat : public CModule
 					Table.SetCell("Host", pSock->GetRemoteIP());
 					Table.SetCell("Port", CString(pSock->GetRemotePort()));
 					SSL_SESSION *pSession = pSock->GetSSLSession();
+#ifdef HAVE_OPAQUE_SSL_SESSION
+					if (pSession)
+						Table.SetCell("Cipher", SSL_CIPHER_get_name(SSL_SESSION_get0_cipher(pSession)));
+#else
 					if (pSession && pSession->cipher && pSession->cipher->name)
 						Table.SetCell("Cipher", pSession->cipher->name);
-
+#endif
+					else
+						Table.SetCell("Cipher", "None");
```

Comment:
According to https://www.openssl.org/news/changelog.html 0.9.7 was in 2005 and earlier. I think that's long enough to not care too much about it.